test/0.8.25/vaults/contracts/Protocol__Deployment.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../../../contracts/0.8.9/EIP712StETH.sol";
+import "forge-std/Test.sol";
+
+import {CommonBase} from "forge-std/Base.sol";
+import {LidoLocator} from "../../../../contracts/0.8.9/LidoLocator.sol";
```

Comment:
```suggestion
import {LidoLocator} from "contracts/0.8.9/LidoLocator.sol";
```

Probably reorder internal imports to single place (top / bottom) for clarity

Reply:
refactored